src/extensions/main-api/power.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { powerMonitor } from "electron";
+import type { Disposer } from "../../common/utils/disposer";
+export type PowerEventListener = () => void;
+
+export const onSuspend = (listener: PowerEventListener): Disposer => {
```

Comment:
Note: in the future we will need to do some documentation for these functions, it might be useful to do that now. Just a simple one liner doc comment explaining what each of these four items (the 3 functions and the type) "do"